### PR TITLE
Delete yamls; update yamls urls and selectors

### PR DESCRIPTION
**Pull request by OdeclasV**

### CODE REVIEW

File: members/I000024.yaml [hunk 1]
```diff
@@ -1,87 +1,13 @@
...
-    - check:
-      - name: response
-        selector: "#response_y"
-        value: "Y"
-    - select:
-      - name: Your Prefix
-        selector: "#salutation"
-        value: "$NAME_PREFIX"
+      - name: Prefix
+        selector: select#input-B03E0527-4040-F985-52CD-C0DD6BEF325F
```
Comment (tirish) [inline members/I000024.yaml]: Are these generated frequently or relatively stable?

File: members/STXLc4e68274.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
-bioguide: TXL000236
+bioguide: STXLc4e68274
 contact_form:
   method: post
-  action: /members/member-page/email/?district=8&session=84
+  action: ""
```
Comment (tirish) [inline members/STXLc4e68274.yaml]: Just FYI, this `action` is not used by any of our stuff (so you can always omit it if you want)

File: members/I000024.yaml (hunk 1 above)
Comment (OdeclasV) [reply members/I000024.yaml]: Hard to say. My read is that they can be considered relatively stable, but can't say for sure. I tried looking for other ways to select these elements on the page but didn't see anything more unique or more stable.

File: members/STXLc4e68274.yaml (hunk 2 above)
Comment (OdeclasV) [reply members/STXLc4e68274.yaml]: 👍

Comment (tirish) [reply members/STXLc4e68274.yaml]: and by omit I mean like leave the line out entirely

**Pull request merged**